bgpd/bgp_fsm.c

```diff
@@ -1310,7 +1310,7 @@ static void bgp_start_deferral_timer(struct bgp *bgp, afi_t afi, safi_t safi,
 	 * for negotiated AFI/SAFI.
 	 */
 	for (ALL_LIST_ELEMENTS(bgp->peer, node, nnode, peer)) {
-		if (peer->connection->status != Established)
+		if (peer_established(peer->connection))
```

Comment:
We switch the logic here, so it was a "dead code" or something?

Reply:
Logic has been updated in commit `bgpd: Further fixes for GR EOR wait`